samples/reactjs-todo/.env.example

```diff
@@ -1,6 +1,5 @@
 AM_URL=$AM_URL
 API_URL=$API_URL
-APP_URL=$APP_URL
```

Comment:
Did you remove this because the API doesn't use this env variable at all?

Reply:
I removed it because we use window.location for this instead so when i copy env variables it doesnt break